pyproject.toml

```diff
@@ -16,20 +16,20 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-aiohttp = {extras = ["speedups"], version = "^3.7.3"}
-pydantic = "^1.7.3"
+python = "^3.11"
+aiohttp = "3.9.0"
```

Comment:
is there a reason this needs to be pinned to exactly 3.9.0?

Reply:
I had to use a specific beta version during a few weeks (3.9.0b0), because of the 3.11 python version I was using.
When it was finally included in the 3.9.0, I forgot to put it back.
I'll move it back to : `aiohttp = {extras = ["speedups"], version = "^3.9.0"}`